.github/ISSUE_TEMPLATE/iceberg_proposal.yml

```diff
@@ -47,4 +47,10 @@ body:
         - label: REST
         - label: Puffin
         - label: Encryption
-        - label: Other
\ No newline at end of file
+        - label: Other
+  - type: checkboxes
```

Comment:
Thanks for your patience here, I was touching grass.

A proposal starts with a Google doc, for example, the Geo support: https://docs.google.com/document/d/1iVFbrRNEzZl8tDcZC81GFt01QJkLJsI9E2NBOt21IRI

Once there is consensus, the process of splitting it out into separate issues starts. I'm doubtful if having this additional checkbox and TODO label clarifies the process.

In addition to that, we started in Python (https://github.com/apache/iceberg-python/issues/736) and Rust (https://github.com/apache/iceberg-rust/issues/348) to have tracking issues as suggested a while ago on the dev-list, which also seem in conflict with having a tracking issue per proposal.

If the tracking issue works for the partitioned stats, I'm fully okay with that, just not sure if we should add this to the template (since it is easy to add a lot of additional checkboxes here, which might raise the bar of proposing something at all).